### ISSUE TITLE
BUG: Closing tag linting issue in VS Studio #9

### ISSUE THREAD

**Issue by jmarshall9120** [johnsoncodehk/volar#1756]
The closing tags are coming up as unrecognized suddenly in templates, and none of the outputs in the console (ESLint, Prettier).  There is no specific output for volar that I see. 

![image](https://user-images.githubusercontent.com/30184747/186255170-6301189b-2a84-4a16-b16a-7c1c82be5f49.png)

I'm digging in.  It looks like Volar requires a jsconfig.json (which vuter did not).  I put that in, but no real change. Any ideas appreciated. 

I'm on:
Windows 11
Volar 0.40.1
Typescript Vue Plugin: 0.40.1 (not typescript in current project howerver)
VS Code: July 2022 (version 1.70)

**Comment by jmarshall9120** [comment]
Update:  I have found the culprit.

see my so post [here](https://stackoverflow.com/questions/73464356/vue-js-closing-tag-linting-is-broken/73464849#73464849), copied below

Okay I figured it out.  As it turns out the the `lang="html"` attribute in the `template` tag must be removed to make this work.  

Funny fact, I actually submitted a ticket to VSCode/Vuter a little while ago where we had broken syntax and determined that a bug made it necessary to include the `lang` tag. 

Odd that the bug has flipflopped, so I guess if you (the reader) happen to be a poor soul experiencing this you will need to try both approaches to determine what your combination of VSCode/Vue language support will accept:

TLDR: 

```js
//used to require this in SFC's:
<template lang="html">
</template>
```

```
// now requires this with volar
<template>
<template>
```

As far as I know the `lang` tag is still HTML standard.  You might consider patching this.
